dags/hivemind_etl_helpers/tests/integration/test_gdrive_ingestion_etl.py

```diff
@@ -44,13 +45,14 @@ def test_run_pipeline(self):
         self.assertEqual(len(processed_result), 2)
 
     def test_load_pipeline_run_exception(self):
-        ingest_pipeline = Mock(IngestionPipeline)
-        community = "1234"
-        self.setUpDB(community)
-        gdrive_pipeline = GoogleDriveIngestionPipeline("1234")
-        gdrive_pipeline.cohere_model = MockEmbedding(embed_dim=1024)
+        gdrive_pipeline = CustomIngestionPipeline(
+            "1234", table_name="gdrive", testing=True
+        )
+        gdrive_pipeline.run_pipeline = Mock()
+        gdrive_pipeline.run_pipeline.side_effect = Exception("Test Exception")
         docs = ["ww"]
-
-        processed_result = gdrive_pipeline.run_pipeline(docs)
-        ingest_pipeline.side_effect = Exception("Test Exception")
-        self.assertIsNone(processed_result)
+        try:
+            gdrive_pipeline.run_pipeline(docs)
+        except Exception as e:
```

Comment:
This is not checking the ingestion pipeline in the right way as any exceptions appear or not the test case will always pass. You can probably do the following
```
with self.assertRaises(Exception):  # you can specify the exception type too
   # pipeline call
```